src/components/reusable/table/table-row.ts

```diff
@@ -197,6 +218,7 @@ export class TableRow extends LitElement {
       )
     ) {
       this.expanded = expanded;
+      this.assistiveText = `${expanded ? 'Expanded' : 'Collapsed'}`;
```

Comment:
We can't hard-code any strings, they must be customizable for localization purposes.

Reply:
Added textStrings prop on kyn-tr